libc-test/build.rs

```diff
@@ -1927,6 +1928,157 @@ fn test_emscripten(target: &str) {
+               "unistd.h",
+               "utime.h",
+               "wchar.h",
+               "errno.h",
+    }
+
+    cfg.skip_struct(move |ty| {
+        match ty {
+            "passwd"      |
+            //the below definitions is part of public/private/ 
+            "_Vx_EVENTS_RSRC" |
+            "_Vx_semaphore" |
+            //Avaliable in Kernel space.
+            "epoll_event" => true,
+            _ => false,
+        }
+    });
+
+    /* Fix me */
+    cfg.skip_const(move |name| match name {
+        "SIG_DFL"
+        | "SIG_IGN"
+        | "SIG_ERR"
+        | "EPOLLPRI"
+        | "EPOLLIN"
+        | "EPOLLOUT"
+        | "EPOLLERR"
+        | "EPOLLHUP"
+        | "EPOLLRDHUP"
+        | "EPOLLONESHOT"
+        | "EPOLLET"
+        | "EPOLL_CTL_ADD"
+        | "EPOLL_CTL_DEL"
+        | "EPOLL_CTL_MOD"
+        | "S_nfsLib_NFSERR_XDEV"
+        | "S_nfsLib_NFSERR_NODEV"
+        | "S_nfsLib_NFSERR_MLINK"
+        | "S_IFDEVMEM"
+        | "CHILD"
+        | "TIME_RELTIME"
+        | "RTLD_DEFAULT"
+        | "EIOA"
+        | "FIONBIO"
+        | "FIOWRITE"
+        | "FIONREAD"
+        | "VXSIM_EWOULDBLOCK" => true,
+        _ => false,
+    });
+
+    /* Fix me */
+    cfg.skip_type(move |ty| match ty {
+        "stat64" | "sighandler_t" | "off64_t" => true,
+        _ => false,
+    });
+
+    cfg.skip_field_type(move |struct_, field| match (struct_, field) {
+        ("siginfo_t", "si_value")
+        | ("stat", "st_size")
+        | ("sigaction", "sa_u") => true,
+        _ => false,
+    });
+
+    cfg.skip_roundtrip(move |s| match s {
+        _ => false,
+    });
+
+    cfg.type_name(move |ty, is_struct, is_union| match ty {
+        "DIR" | "FILE" | "Dl_info" | "RTP_DESC" => ty.to_string(),
+        t if is_union => format!("union {}", t),
+        t if t.ends_with("_t") => t.to_string(),
+        t if is_struct => format!("struct {}", t),
+        t => t.to_string(),
+    });
+
+    /* Fix me */
+    cfg.skip_fn(move |name| match name {
+        "sigqueue" | "_sigqueue" | "_rtld_dladdr" | "getpwnam"
+        | "getpwuid" | "pclose" | "execl" | "execle" | "execlp" | "execv"
+        | "execve" | "renameat" | "readlinkat" | "mkdirat" | "linkat"
+        | "fchownat" | "fchmodat" | "openat" | "fdopendir" | "symlinkat"
+        | "lchown" | "setpgid" | "fstatat" | "setsid" | "tcgetpgrp"
+        | "tcsetpgrp" | "killpg" | "mlock" | "munlock" | "mlockall"
+        | "munlockall" | "mmap" | "munmap" | "if_nametoindex"
+        | "if_indextoname" | "flock" | "res_init" | "mknod" | "chroot"
+        | "getsid" | "tcdrain" | "tcflow" | "tcflush" | "tcgetsid"
+        | "tcsendbreak" | "mkdtemp" | "nice" | "grantpt" | "posix_openpt"
+        | "ptsname" | "unlockpt" | "strcasestr" | "socketpair"
+        | "pwrite64" | "pread64" | "fork" | "_pathIsAbsolute" | "execvp"
+        | "setgroups" | "signal" | "getpagesize" | "dlclose" | "dlsym"
+        | "dlerror" | "dlopen" | "setrlimit" | "getrlimit" => true,
```

Comment:
Please indicate how this list of functions was identified.

Reply:
@UmeshKalappa 
Can you help to comment on this question?

Reply:
This is error I get after removing "sigqueue", I believe others have such similiar issue:

running: "wr-c++" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-m64" "-static" "-Wall" "-We\
xtra" "-Wall" "-Wextra" "-Werror" "-Wno-unused-parameter" "-Wno-type-limits" "-Wno-address-of-packed-member" "-Wno-unknown-warning-o\
ption" "-Wno-deprecated-declarations" "-o" "/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/build/l\
ibc-test-4e44ace510e99266/out/main.o" "-c" "/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/build/l\
ibc-test-4e44ace510e99266/out/main.c"
cargo:warning=/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/build/libc-test-4e44ace510e99266/out/\
main.c:11858:57: error: type specifier missing, defaults to 'int' [-Werror,-Wimplicit-int]
cargo:warning=            int (*__test_fn_sigqueue(void))(pid_t, int, sigval) {
cargo:warning=                                                        ^
cargo:warning=/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/build/libc-test-4e44ace510e99266/out/\
main.c:11859:24: error: incompatible pointer types returning 'int (pid_t, int, const union sigval)' (aka 'int (int, int, const union\
 sigval)') from a function with result type 'int (*)(pid_t, int, int)' (aka 'int (*)(int, int, int)') [-Werror,-Wincompatible-pointe\
r-types]
cargo:warning=                return sigqueue;
cargo:warning=                       ^~~~~~~~
cargo:warning=2 errors generated.
exit code: 1

Reply:
Ok. Is this list is captured somewhere in a defect or something so we have a record of which ones were skipped? If we don't have a tracker issue, we won't ever re-visit it.

Reply:
Yes, these skipped cases are tracked with this record:
https://jira.wrs.com/browse/V7LIBC-967